openupgrade_scripts/scripts/mail/15.0.1.5/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,266 @@
+mail         / mail.guest               / timezone (selection)          : NEW selection_keys: function
+# NOTHING TO DO: new feature
+
+mail         / mail.ice.server          / credential (char)             : NEW
+mail         / mail.ice.server          / server_type (selection)       : NEW required, selection_keys: ['stun', 'turn'], req_default: function, hasdefault
+mail         / mail.ice.server          / uri (char)                    : NEW required
+mail         / mail.ice.server          / username (char)               : NEW
+# NOTHING TO DO: new feature call audio and video
+
+mail         / mail.mail                / failure_type (selection)      : NEW selection_keys: ['mail_bl', 'mail_dup', 'mail_email_invalid', 'mail_email_missing', 'mail_optout', 'mail_smtp', 'unknown']
+# NOTHING DO DO
+
+mail         / mail.mail                / is_notification (boolean)     : NEW
+mail         / mail.mail                / notification (boolean)        : DEL
+# DONE: pre-migration: rename fields notification -> is_notification
+
+mail         / mail.message             / author_guest_id (many2one)    : NEW relation: mail.guest
+# NOTHING TO DO: new feature
+
+mail         / mail.message             / channel_ids (many2many)       : DEL relation: mail.channel
+# NOTHING TO DO: feature removed
+
+mail         / mail.message             / moderation_status (selection) : DEL selection_keys: ['accepted', 'pending_moderation', 'rejected']
+mail         / mail.message             / moderator_id (many2one)       : DEL relation: res.users
+# NOTHING TO DO: transfer data to model mail_group_message
+
+mail         / mail.message             / notified_partner_ids (many2many): table is now 'mail_notification' ('mail_message_res_partner_needaction_rel')
+# DONE: pre-migration: rename table
+
+mail         / mail.message             / reaction_ids (one2many)       : NEW relation: mail.message.reaction
+# NOTHING TO DO: new feature
+
+mail         / mail.message             / no_auto_thread (boolean)      : DEL
+mail         / mail.message             / reply_to_force_new (boolean)  : NEW
+# DONE: pre-migration: rename fields no_auto_thread -> reply_to_force_new
+
+mail         / mail.message.reaction    / content (char)                : NEW required
+mail         / mail.message.reaction    / guest_id (many2one)           : NEW relation: mail.guest
+mail         / mail.message.reaction    / message_id (many2one)         : NEW relation: mail.message, required
+mail         / mail.message.reaction    / partner_id (many2one)         : NEW relation: res.partner
+# NOTHING TO DO: new feature
+
+mail         / mail.moderation          / channel_id (many2one)         : DEL relation: mail.channel, required
+mail         / mail.moderation          / email (char)                  : DEL required
+mail         / mail.moderation          / status (selection)            : DEL required, selection_keys: ['allow', 'ban']
+# DONE: transfer data to model mail_group_moderation
```

Comment:
Did you actually do anything for this, or is this to be done by the person to undertake the migration of mail_group?

Reply:
As discussed above, we don't need migration for this change. I forgot to edit the description, this should be nothing to do.